### ISSUE TITLE
Patch vulnerabilities

### ISSUE THREAD

**Issue by dansvend** [equinor/MAD-VSM-WEB#456]
Patch vulnerabilities found by Snyk.

**Comment by dansvend** [comment]
@Sayyare 

This is ready for testing 👩‍🔬

```
-----------------------------
| Platform     | Web        |
| Version      | 0.2.21     |
| Commit | f5f76d9 |
| Environment  | Test       |
-----------------------------
```

**Comment by Sayyare** [comment]
Tested Fly Web App in test environment, working good as expected. 🐱‍👤
The user is able to share/add other user to the existing process, and both user are able to update and see the updated contents. Deep link, feedback and other features are working good as well. @dansvend 
Moving the task to QA.

**Comment by dansvend** [comment]
Thank you so much for testing 🙏 Good job, @Sayyare 💪 

This has been built to QA and is ready for testing 👩‍🔬

```
-----------------------------
| Platform     | Web        |
| Version      | 0.2.21     |
| Commit       | f5f76d9    |
| Environment  | QA       |
-----------------------------
```

**Comment by dansvend** [comment]
Thanks for testing @Sayyare 🙏 

We're ready to deploy upgrade from ADAL to MSAL for the authentication on the API and a security patch to the frontend.

Our tester, @Sayyare , have tested the changes in both test and QA environments and it's working as expected, but it never hurts having more eyes on it. Would you like to do some testing in the QA environment as well @askbulle and @knudsvik before we release? 🚀

**Comment by knudsvik** [comment]
LGTM (not too sure what I am looking for, but the app behaviour is as expected) :)

**Comment by dansvend** [comment]
Thanks for testing @knudsvik 👌

New authentication method have been implemented in the API in addition to packages being upgraded to newer versions to patch vulnerabilities.

If everything seems to work as expected we can release this to production after coordinating this with @anheri so that we can deploy both the API and frontend. This will mean there will be some downtime when we release.

**Comment by dansvend** [comment]
This has been released to production in 0.2.22 the 11th January 2022🚀 I'm moving it to `✅ Done`.
